### ISSUE TITLE
[rc-0.5.0] Balance/Pending Tx Issues

### ISSUE THREAD

**Issue by sdtsui** [kyokan/bob-wallet#205]
Context: Low-usage wallet, <10 names in it.

I notice that when someone sends me funds, it doesn't catch new transactions automatically.  Leaving my computer open when connected to RPC basically seems to do nothing -- I can see stuff confirm on HNScan, but I only see the tx get seen **after** a refresh (ctrl+r) or full restart of the app.

In addition, after the reset/refresh, I notice that the Spendable Balance number doesn't update -- like I can see the inbound funds confirmed in Transaction History, but SB doesn't change until I do yet another refresh.

In summary, I've never seen a pending tx...it seems to be skipping that and only showing received funds.  The expected behavior would be automatically catching Pending txs and UI consistency when updating balances.

Note that this is just for sending funds. I haven't paid as much attention today to what it does when sending bids.
